### DISCUSSION TITLE
Is license `AGPL-3.0-or-later` or `AGPL-3.0`?

### DISCUSSION THREAD

**Discussion by eNV25** [koreader/koreader#11652] [Q&A] [answered]
spdx has two different IDs for AGPLv3:

https://spdx.org/licenses/AGPL-3.0.html

https://spdx.org/licenses/AGPL-3.0-or-later.html

**Comment by Frenzie** [comment, marked as answer]
The way I interpret it is that it's v3 or any later version (since afaik there's no statement to the contrary).

https://github.com/koreader/koreader/blob/31c28378e7d9566969697968f6c745b14089be2a/COPYING#L561-L585
